octopoes/bits/definitions.py

```diff
@@ -27,6 +27,7 @@ class BitDefinition(BaseModel):
     min_scan_level: int = 1
     default_enabled: bool = True
     config_ooi_relation_path: str | None = None
+    cache_lifetime: int = 3600
```

Comment:
I think a `timedelta` is a better type than an `int` for a duration field. `PositiveInt` could also work, but it's less descriptive in this case

Reply:
See a9ff441948996c1e6d66a3464c5d5e2f95088da9